charts/prometheus-memcached-exporter/values.yaml

```diff
@@ -39,6 +39,10 @@ securityContext: {}
 service:
   type: ClusterIP
   port: 9150
+  # Labels to append to the service
+  extraLabels: {}
```

Comment:
It is quite common in the repository's charts to name the field in this context just `labels`. Would it be possible to rename `extraLabels`?

Reply:
sure thing :)